### PR TITLE
add new platform fields

### PR DESCRIPTION
**Pull request by ntlong1594**

### CODE REVIEW

File: src/engine/Api/V1/Parsers/Appointments.php [hunk 1]
```diff
@@ -36,7 +36,8 @@ public function encode(array &$response)
             'customerId' => $response['id_users_customer'] !== NULL ? (int)$response['id_users_customer'] : NULL,
             'providerId' => $response['id_users_provider'] !== NULL ? (int)$response['id_users_provider'] : NULL,
             'serviceId' => $response['id_services'] !== NULL ? (int)$response['id_services'] : NULL,
-            'googleCalendarId' => $response['id_google_calendar'] !== NULL ? (int)$response['id_google_calendar'] : NULL
+            'googleCalendarId' => $response['id_google_calendar'] !== NULL ? (int)$response['id_google_calendar'] : NULL,
+            'platform' => $response['platform']
```
Comment (chicanh) [inline src/engine/Api/V1/Parsers/Appointments.php]: add to v2, if v3 is available please add to v3 also
to keep this unchanged

File: src/assets/sql/structure.sql [hunk 2]
```diff
@@ -389,4 +389,7 @@ ALTER TABLE `integrated_users_patients`
 
 
 ALTER TABLE `integrated_users_patients` DROP PRIMARY KEY, ADD PRIMARY KEY(`id_user_integrated`, `id_patients`);
-ALTER TABLE `integrated_users_patients` modify `id_service_integrated` VARCHAR(50);
\ No newline at end of file
+ALTER TABLE `integrated_users_patients` modify `id_service_integrated` VARCHAR(50);
+
+
+ALTER TABLE `ea_appointments` ADD COLUMN `platform` VARCHAR(255);
```
Comment (chicanh) [inline src/assets/sql/structure.sql]: VARCHAR(15)

Comment (ntlong1594) [reply src/assets/sql/structure.sql]: updated

File: src/engine/Api/V1/Parsers/Appointments.php (hunk 1 above)
Comment (ntlong1594) [reply src/engine/Api/V1/Parsers/Appointments.php]: updated

**Review by chicanh — APPROVED** [review]

**Pull request merged**